File: Parksujin/[BOJ] 소트인사이드.py
N = int(input())
num = list(map(int, str(N))) # str형인 N을 다시 int로 바꾼 후 list에 저장
num.sort(reverse = True)
for i in range(len(num)):
  print(num[i], end = '') # end = ''를 통해 한줄로 출력

File: Kanghyun/[BOJ] 소트인사이드.cpp
#include<iostream>
#include<vector>
#include<algorithm>
using namespace std;
int main() {
	int N, divider = 1;
	cin >> N;
		vector<int> num;
		for (int i = 0; i < 10; i++) {
			if (N / divider == 0)
				break;
			num.push_back((N / divider) % 10);
			divider *= 10;
		}
		sort(num.begin(), num.end(), greater<>());
		for (int i = 0; i < num.size(); i++)
			cout << num[i];
}